BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -371,12 +375,21 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
             }
         }
     }
+    
+    private func applySnapshot() {
+        let goalsToShow = versionManager.lastChckedUpdateState() == .UpdateRequired ? [] : filteredGoals
+        
+        var snapshot = NSDiffableDataSourceSnapshot<Section, Goal>()
+        snapshot.appendSections([.main])
+        snapshot.appendItems(goalsToShow, toSection: .main)
```

Comment:
```suggestion
        snapshot.appendItems(goalsToShow, toSection: .main)
        snapshot.reconfigureItems(goalsToShow)
```
Without this, the view does not pick up when existing goals are changed - it sees the members of the list are the same as before, and doesn't realize their contents may have changed.